mycroft/configuration/mycroft.conf

```diff
@@ -111,14 +111,28 @@
     "sample_rate": 16000,
     "channels": 1,
     "record_wake_words": false,
-    "wake_word": "hey mycroft",
-    "phonemes": "HH EY . M AY K R AO F T",
```

Comment:
Not using this key will ignore server changes. Can you add support for using this value. Otherwise, anyone who's customized it will lose the setting.

Reply:
added support and TODO flag so it can be removed later